hack/install_deps_rpm.sh

```diff
@@ -48,11 +48,13 @@ UNKNOWN="koji rpmdevtools nodejs-packaging"
 
 GOLANG_SUPPORT="golang"
 
+# for pcp pmcd metrics collector
+PCP="pcp"
```

Comment:
Uhm... does anybody remember why we introduced this `hack/install_deps_rpm.sh` instead of installing the RPM dependencies directly in Dockerfile?

Reply:
To divide big `Dockerfile` into logical parts and thus make it more readable. Thanks for merging.